build/NI.CSharp.Analyzers.nuspec

```diff
@@ -16,7 +16,7 @@
     <license type="file">LICENSE.txt</license>
     <dependencies>
       <group targetFramework="netstandard2.0">
-        <dependency id="Microsoft.CodeAnalysis.NetAnalyzers" version="7.0.4" />
+        <dependency id="Microsoft.CodeAnalysis.NetAnalyzers" version="8.0.0" />
```

Comment:
Maybe we should add a comment here to update `AnalysisLevel` in NI.CSharp.Analyzers.props so they don't get out of sync

Reply:
Good point, comment added.